index.html

```diff
@@ -10,6 +10,113 @@
+        </div>
+        <nav>
+            <ul class="nav-links">
+                <li class="link">Meet Karma</li>
+                <li class="link">How it Works</li>
+                <li class="link">Store</li>
+                <li class="link">Blog</li>
+                <li class="link">Help</li>
+                <li class="link">Login</li>
+            </ul>
+        </nav>
+        <div class="hamburger">
+            <svg width="24" height="16" viewBox="0 0 24 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path fill-rule="evenodd" clip-rule="evenodd" d="M0 1C0 0.447715 0.447715 0 1 0H23C23.5523 0 24 0.447715 24 1V1.28572C24 1.838 23.5523 2.28572 23 2.28572H1C0.447716 2.28572 0 1.838 0 1.28572V1ZM0 7.85714C0 7.30485 0.447715 6.85713 1 6.85713H23C23.5523 6.85713 24 7.30485 24 7.85713V8.14285C24 8.69513 23.5523 9.14285 23 9.14285H1C0.447716 9.14285 0 8.69513 0 8.14285V7.85714ZM1 13.7143C0.447715 13.7143 0 14.162 0 14.7143V15C0 15.5523 0.447716 16 1 16H23C23.5523 16 24 15.5523 24 15V14.7143C24 14.162 23.5523 13.7143 23 13.7143H1Z" fill="black"/>
+                </svg>
+        </div>
+    </header>
+    <main>
+        <section class="intro">
+            <!--this div is halpful to put all the text at center of the intro div-->
+            <div>
+                <h1>Introducing Karma</h1>
+                <p>Bring WiFi with you, everywhere you go.
+                </p>
+                <button class="orange">Learn More</button>
+            </div>
+            
+        </section>
+        <section class="services">
+            <div class="title">
+                <h2>Everyone needs little Karma</h2> 
+            </div>
+            <div class="services-container">
+                <div class="individual-service-box grow">
+                    <svg class="service-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128">
+                        <path fill="#F05A29"
+                            d="M122 66.463h-1.106v-48.41c0-3.033-2.468-5.5-5.5-5.5H13.521a5.506 5.506 0 00-5.5 5.5v38.126h-2.02a5.506 5.506 0 00-5.5 5.5v51.996c0 3.032 2.467 5.5 5.5 5.5h35.546c3.033 0 5.5-2.468 5.5-5.5V90.627H57.69v10.04h-5.422a1.5 1.5 0 100 3H81.5a1.5 1.5 0 000-3h-5.421v-10.04h23.944v16.562c0 3.032 2.468 5.5 5.5 5.5H122c3.032 0 5.5-2.468 5.5-5.5V71.963c0-3.033-2.468-5.5-5.5-5.5zm-77.953 47.212c0 1.379-1.122 2.5-2.5 2.5H6.001a2.503 2.503 0 01-2.5-2.5v-4.286h40.546v4.286zm0-7.286H3.501v-44.71c0-1.378 1.122-2.5 2.5-2.5l35.546-.001c1.378 0 2.5 1.122 2.5 2.5v44.711zm29.032-5.722h-12.39v-10.04h12.39v10.04zm-26.032-13.04v-9.548h52.976v9.548H47.047zm52.976-15.664v3.116H47.047V61.678c0-3.033-2.467-5.5-5.5-5.5l-30.525.001V18.053c0-1.378 1.122-2.5 2.5-2.5h101.873c1.379 0 2.5 1.122 2.5 2.5v48.41h-12.371a5.507 5.507 0 00-5.501 5.5zm24.477 35.226c0 1.379-1.121 2.5-2.5 2.5h-16.477a2.502 2.502 0 01-2.5-2.5v-4.08H124.5v4.08zm0-7.08h-21.477V71.963c0-1.379 1.121-2.5 2.5-2.5H122c1.379 0 2.5 1.121 2.5 2.5v28.146z" />
+                        </svg>
```

Comment:
Not sure why this is an svg as I think you were given the images? It is important to use images as without the images you can not add alt tags which are important for many reasons, mainly for users using a screen reader to know what the image is without seeing it